### ISSUE TITLE
Error: fatal: bad revision

### ISSUE THREAD

**Issue by MilanDasek** [weaveworks/flux#1879]
Hello,

When I try 
```
fluxctl --k8s-fwd-ns=flux sync
```

I got
```
Synchronizing with git@bitbucket.org:XXX/can-docker-gitops.git
HEAD of master is 3c5f564
Waiting for 3c5f564 to be applied ...
== Error ==

Error: fatal: bad revision 'eks-staging-can..3c5f564'

We don't have a specific help message for the error above.
```

and
```
ts=2019-03-29T12:28:40.950134623Z caller=loop.go:90 component=sync-loop err="exit status 129"
```

when I try to add a tag: 
```
git tag -a -f eks-staging-can %ID%
git push --tags
```

then I got 
```
ts=2019-03-29T12:14:48.345991395Z caller=loop.go:90 component=sync-loop err="fatal: ambiguous argument 'eks-staging-can': both revision and filename"
```

I gave in the config:
```
--git-path=common
- --git-label=eks-staging-can
```

I also tried to delete Flux and deploy again - same result.

Can you please advise?

Thx M

**Comment by hiddeco** [comment]
This is a known issue when you have a file in git that is the same as your tag, it has been resolved in #1829 but has not made it into a release yet.

The current workaround would be to change your `--git-label` to something that is not a file in git.

**Comment by MilanDasek** [comment]
ah, ok thanks

I have changed label and it started working

suggestion - please update documentation here: https://github.com/weaveworks/flux/blob/master/site/faq.md

```
--set git.label=cluster-name
--set git.path=k8s/cluster-name
```

if you do, you will end up in my situation
